src/executor.cpp

```diff
@@ -1038,4 +1045,41 @@ void Executor::execute74()
     displayExecutionStatus(74, std::vector<types::FunctionNumber>(), true);
 }
 
-} // namespace panel
\ No newline at end of file
+types::FunctionExecutionResult
+    Executor::executeFuncViaDbus(const types::FunctionNumber funcNum)
+{
+    isExternallyTriggered = true;
+    try
+    {
+        switch (funcNum)
+        {
+            case 21:
+            case 22:
+            case 34:
+            case 65:
+            case 66:
+            case 67:
+            case 68:
+            case 69:
+            case 70:
+                sendFuncNumToPhyp(funcNum);
+                break;
+
+            default:
+                std::cerr << "Given function number "
+                          << static_cast<int>(funcNum)
+                          << " is not supported to execute via d-bus."
+                          << std::endl;
+                executionStatus = std::make_tuple(false, "", "");
+        }
+    }
+    catch (std::exception& e)
+    {
+        executionStatus = std::make_tuple(false, "", "");
+        std::cerr << e.what() << std::endl;
+    }
+
+    isExternallyTriggered = false;
```

Comment:
I don't see it used anywhere between true and false, so setting/inset is required?

Reply:
It is being used in function where we display.

Reply:
Okay, then may be it should be unset there itself.

Reply:
What difference it will make? Set and unset in the same function will keep it easy to track.